src/runtime_src/core/tools/xbutil2/OO_P2P.cpp

```diff
@@ -62,6 +62,19 @@ string2action(std::string str)
     throw xrt_core::generic_error(EINVAL, "Invalid p2p action '" + str + "'");
 }
 
+std::string
+print_success_msg(action_type action)
```

Comment:
This is not a case where we should be encapsulating print messages.

In addition, these types of methods should NOT be throwing an exception.